src/styles/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$calendarWidth: 720px;
+$daySize: 100px;
+$dayGap: 1px;
+$calendarPadd: 10px;
+$month-length: 31;
+$start-day-week: 1 mon, 2 tue, 3 wed, 4 thu, 5 fri, 6 sat, 7 sun;
```

Comment:
'$start-day-week' variable name should follow the camel case naming convention. Consider renaming it to '$startDayWeek'. Furthermore, SCSS doesn't support this kind of value assignment. If you want to map days to numbers, consider using a map data structure.